account.go

```diff
@@ -351,13 +351,11 @@ func (account *Account) prepFunctionInvokeRPC(ctx context.Context, messageType s
 		if err != nil {
 			return nil, err
 		}
-		return &rpc.BroadcastedInvokeV1Transaction{
-			BroadcastedTxnCommonProperties: rpc.BroadcastedTxnCommonProperties{
-				MaxFee:    maxFeeFelt,
-				Version:   version,
-				Signature: []*felt.Felt{s1Felt, s2Felt},
-				Nonce:     nonceFelt,
-			},
+		return &rpc.InvokeTxnV1{
```

Comment:
Probably, although I think this is out of scope of this PR for two reasons 1) the goal was to replace broacastInvokeTxn with InvokeTxn types (which this PR does) not fix the logic in account.go, and 2) this account file is broken, and will be replaced with the new accounts package.

Reply:
I thought to point it out since `Type` was a required field for the `InvokeTxnV1` type, but you are right, this file is already broken and will be replaced soon